src/macaron/__main__.py

```diff
@@ -147,10 +169,16 @@ def analyze_slsa_levels_single(analyzer_single_args: argparse.Namespace) -> None
             logger.error("Error while loading the input provenance file: %s", error)
             sys.exit(os.EX_DATAERR)
 
+    if analyzer_single_args.skip_deps:
+        logger.warning(
+            "The --skip-deps flag has been deprecated and WILL NOT do anything. "
+            + "Dependency resolution is off by default. This flag will be removed soon."
```

Comment:
There is a slight inconsistency between the warning message here and the message in the command line help:
```
Dependency resolution is off by default. This flag does nothing and will be removed in the next release.
```
They should probably just have the same text.

Reply:
I agree. I changed one of them at one point and forgot to update the second one.

Reply:
Fixed in https://github.com/oracle/macaron/pull/840/commits/e880895f6f43889384d7a348754bfca2a93a2d82